src/dataset-builder/SummaryCharts.ts

```diff
@@ -0,0 +1,150 @@
+export function generateAgeChartOptions() {
+  return {
+    categories: [
+      { short: '0-17' },
+      { short: '18-29' },
+      { short: '30-39' },
+      { short: '40-49' },
+      { short: '50-59' },
+      { short: '60-69' },
+      { short: '70-79' },
+      { short: '80+' },
+    ],
+    series: [{ data: [9, 25, 15, 17, 14, 9, 6, 5] }],
+    title: 'Age',
+    yTitle: 'OVERALL PERCENTAGES',
+    height: '400rem',
+    legendEnabled: false,
+    showSeriesName: false,
+    type: 'bar',
+  };
+}
+
+export function generateGenderChartOptions() {
+  return {
+    categories: [
+      { short: 'Man' },
+      { short: 'Woman' },
+      { short: 'Nonbinary' },
+      { short: 'Transgender' },
+      { short: 'Other' },
+      { short: 'Prefer not to Say' },
+    ],
+    series: [
+      {
+        colorByPoint: true,
+        data: [28, 31, 11, 10, 6, 14],
+      },
+    ],
+    title: 'Gender Identity',
+    yTitle: 'OVERALL PERCENTAGES',
+    height: '400rem',
+    legendEnabled: false,
+    showSeriesName: false,
+    type: 'column',
+  };
+}
+
+export function generateRaceChartOptions() {
+  return {
+    categories: [
+      { short: 'White' },
+      { short: 'Black, African American' },
+      { short: 'Hispanic or Latino' },
```

Comment:
I know its like this on the mocks, but I don't think Hispanic/Latino is generally considered a race? I'm not sure what race distinctions we want to use. On the cohort editor page, I used the race distinctions from the census and maybe we would want to use the same ones here? Really I think we would want to use the distinctions that exist in the dataset, so it probably doesn't really matter now as its mocked data.

Reply:
Yeah, maybe we could just double check with Lou what he wants and specifically if he wants Hispanic/Latino listed as a race?

Reply:
Sounds good, I'll message him (and Victoria if she has any thoughts/opinions on this)

Reply:
They agree, better to have the data match - I'll rewrite it!